js/libs/core.js

```diff
@@ -489,10 +489,76 @@ Fliplet.FormBuilder = (function() {
         return !_.isEmpty(this.errors);
       };
 
+      component.methods._hasDuplicateOptions  = function(options) {
+        var finalOptions = _.cloneDeep(options);
```

Comment:
You can clone and map the options all in one go

```suggestion
        options = _.map(options, function(option) {
          return {
            id,
            label
          };
        });
```